### ISSUE TITLE
Question about the parameter historyKeyframeFitnessScore

### ISSUE THREAD

**Issue by Xiangzhaohong** [gisbi-kim/SC-LIO-SAM#2]
you set the parameter  historyKeyframeFitnessScore: 1.2 
But, it may bring some wrong loop factor, why you choose 1.2 ?
 thanks a lot!

**Comment by gisbi-kim** [comment]
no particular reason for that :)
but I just wanna use a slightly larger value than the original repo,
because I found some actually true loops sometimes have ~1 fitness scores so I didn't wanna miss those loops.

If wrong loop factors occured, it's okay because we use a robust kernel for loop factors, 
(see below lines)
https://github.com/gisbi-kim/SC-LIO-SAM/blob/6829a5ba3104096d43fd30233929d5e0b821b4eb/SC-LIO-SAM/src/mapOptmization.cpp#L790
so we can reduce those wrong loops effects when enough true loops exist. 

ps. empirically, I tested the harshest cases where the number of wrong loops was maximized when the scan context descriptor threshold = 1.0 (i.e., all loops were accepted). below is the result and the trajectory still looks good.
 
![cauchy_good1](https://user-images.githubusercontent.com/14989535/102861431-00c88080-4473-11eb-8262-59dcf7ce81b0.png)
